### PR TITLE
CFY-7619 Agents RPM spec

### PR DESCRIPTION
**Pull request by tehasdf**

### CODE REVIEW

**Comment by tehasdf** [comment]
There should be a better way instead of curl in the spec, but i'm not yet sure what is it

File: packaging/cloudify-agents.spec [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+BuildRequires:  python >= 2.7
+Requires(pre):  shadow-utils
+
+
+%description
+Cloudify Agent packages
+
+
+%prep
+curl https://raw.githubusercontent.com/cloudify-cosmo/cloudify-versions/master/packages-urls/agent-packages.yaml > /tmp/agents-list.txt
```
Comment (funkyHat) [inline packaging/cloudify-agents.spec]: If you don't like `curl` here it should be possible to specify this URL as `Source0:` and have `build_rpm.py` download it to ${RPM_SOURCE_DIR} for you

Comment (funkyHat) [reply packaging/cloudify-agents.spec]: & you can then refer to it as `%{S:0}` (see cloudify-premium's spec for an example)

Comment (funkyHat) [reply packaging/cloudify-agents.spec]: I PRd your PR https://github.com/cloudify-cosmo/cloudify-manager/pull/1146

**Pull request merged**